server/gqlcorp/mutation.js

```diff
@@ -31,6 +31,16 @@ const convertTypesforMutation = object => {
         case "date":
           object[table][column] = "String";
           break;
+        case 'numeric':
+          object[table][column] = 'Int'
+        break;
+        case 'smallint':
+          object[table][column] = 'Int'
+          break;
+        case 'ARRAY':
+          object[table][column] = '|String|'
```

Comment:
not sure if specific enough to handle all array cases in long run; they are custom in gql